models/articles.models.js

```diff
@@ -0,0 +1,23 @@
+const db = require("../db/connection");
+
+exports.findArticle = (article_id) => {
+  let queryString = ` SELECT articles.* FROM articles 
+    JOIN topics ON topics.slug = articles.topic
+    JOIN users ON users.username = articles.author `;
+  const queries = [];
+
+  if (article_id) {
+    queryString += `WHERE article_id = $1`;
+    queries.push(article_id);
+  }

```

Comment:
With this model, you've currently got logic in here to decide whether or not the user is getting all of the articles, or just a single article. The following task is going to be getting all articles, so that should be in its own model and controller. At this point, you've essentially done task 4 and 5 in the same model and controller if that makes sense?